scripts/move_to_scala_2.10.sh

```diff
@@ -2,5 +2,17 @@
 
 set +x
 
-find . -name "pom.xml" -exec sed -e "s/2.11.8/2.10.6/g" -e "s/2.11/2.10/g" -i.2.10.bak '{}' \;
-find . -name "*.2.10.bak" -exec rm {} \;
+grep "<scala\.version>" pom.xml | grep -q 2.10
+if [[ $? == 0 ]];
+then
+    echo "Scala version is already set to 2.10 (Scala artifacts have _2.10 version suffix in artifact name)."
+    echo "Cowardly refusing to move to Scala 2.10 a second time..."
```

Comment:
I think I misunderstood you the first time around. I thought that what you were saying was that the previous messages weren't sufficiently descriptive, so I added to them in https://github.com/bigdatagenomics/adam/pull/1319/commits/bdc90a89765a6542a44650980f79022ab0987463. E.g., here we went from:

```
    echo "Scala version is already set to 2.10."
     echo "Cowardly refusing to move to Scala 2.10."
```

to:

```
    echo "Scala version is already set to 2.10 (Scala artifacts have _2.10 version suffix in artifact name)."
    echo "Cowardly refusing to move to Scala 2.10 a second time..."
```

Since these messages are still not looking good to you, I think I must've misunderstood your critique. If you've got a minute, perhaps it'd be useful if you gave an example of what you think a good message would be?